activestorage/app/models/active_storage/attachment.rb

```diff
@@ -36,7 +36,13 @@ def identify_blob
     end
 
     def analyze_blob_later
-      blob.analyze_later unless blob.analyzed?
```

Comment:
Would think this would just be `blob.analyze_later if blob.analyzable? && !blob.analyzed?`?